src/main/resources/defaults/TestGenie.properties

```diff
@@ -14,4 +14,3 @@ buildPath=target/classes
 showCoverage=true
 buildCommand=mvn compile
 telemetryEnabled=false
-telemetryDirectory=TestGenieTelemetry
```

Comment:
I think that it is still better to keep this prefix so that we generate the files inside a directory, rather than just dump all the files by default in the home directory?
Maybe even make this directory hidden (aka starting with a dot)?

Reply:
I will let @martinmladenov decide that one.

Reply:
I think it doesn't really matter, the user is going to change the path anyways :)